data_collection/gazette/spiders/base/portal_da_transparencia.py

```diff
@@ -0,0 +1,54 @@
+import datetime
+import re
+
+import dateparser
+from scrapy import Request
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class PortalDaTransparenciaBaseSpider(BaseGazetteSpider):
+    allowed_domains = ["portaldatransparencia.com.br"]
+    start_date = datetime.date(2000, 1, 1)
+
+    def parse(self, response):
+        EDITION_QUERY = "div.box-edicao"
+        for issue in response.css(EDITION_QUERY):
```

Comment:
**suggestion**: I don't see any gain on  creating this constant just to use it in the next line. If the value is not used in a different location, it just add some context change when we are reading the code. Also, if the webpage changes its content, it is very unlikely that we will just need to update this specific selector to fix it (as probably the entire page changed).

```
editions = response.css("div.box-edicao")
for edition in editions:
    ....
```
or simply:
```
for issue in response.css("div.box-edicao"):
    ...
```

Reply:
I like to declare these constants so they inform the reader what it's meant for (in this case, get the edition data). Becomes a bit less of a "magic string"

Reply:
So `editions = response.css("div.box-edicao")` makes more sense in my opinion. It clearly indicates what is the purpose of the selector content. What bothers me most on this approach is that I need to change the context (move my focus to a different line) to see what exactly I am searching. Considering that we expect to have selector strings in a spider code, they are not magic strings, they are just part of the process to get the information. AS you are not reusing the string in other places, it makes no much sense to have an extra variable just for that.

Reply:
Thanks 👍